src/NuGet.Core/NuGet.Protocol.Core.v3/Constants.cs

```diff
@@ -37,31 +37,31 @@ public static class ServiceTypes
 
     public static class Properties
     {
-        public static readonly string SubjectId = "@id";
-        public static readonly string Type = "@type";
+        public const string SubjectId = "@id";
```

Comment:
Const needed for JsonProperty annotations. I'll go ahead and make it clear these are strictly json parser constants.